pkg/process/manager.go

```diff
@@ -296,6 +297,20 @@ func (m *Manager) Reconnect(ctx context.Context) error {
 		info, err := d.CheckStatus()
 		if err != nil {
 			log.L.WithField("daemon", d.ID).Warnf("failed to check daemon status: %v", err)
+
+			if d.ID == daemon.SharedNydusDaemonID {
```

Comment:
Why only handle the shared daemon case?

Reply:
In fact, multiple daemons mod also has this issue. A following PR is going to resolve it

Reply:
> Why only handle the shared daemon case?

In fact, multiple daemons mod also has this issue. A following PR is going to resolve it